tests/unit/DateTime/DifferenceTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Coduo\PHPHumanizer\Tests\DateTime;
+
+use PHPUnit_Framework_TestCase;
+
+/**
+ * @author       Ezra Pool <ezra@tsdme.nl>
+ */
+class DifferenceTest extends PHPUnit_Framework_TestCase
+{
+    /**
+     * @test
+     * @dataProvider differenceDataProvider
+     *
+     * @param $expected
+     * @param $date1
+     * @param $date2
+     * @param $instance
+     * @param $is_past
+     */
+    public function it_calculates_differences_correctly($expected, $date1, $date2, $instance, $is_past )
+    {
+        $diff = new \Coduo\PHPHumanizer\DateTime\Difference($date1, $date2);
```

Comment:
Could you please move FQCN into use statement and use only ``$diff = new Difference($date1, $date2);`` here? 
Also please use only camel case in variable names. ``$isPast``